reader/src/main/scala-2.13/za/co/absa/atum/reader/implicits/zio.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.reader.implicits
+
+import sttp.client3.impl.zio.RIOMonadAsyncError
+
+object zio {
+  implicit val ZIOMonad: RIOMonadAsyncError[Any] = new RIOMonadAsyncError[Any]
```

Comment:
Please rename this. It's confusing. I would personally stick to its name `rioMonadAssyncError`.